hadoop-mr-s3/src/main/java/org/apache/hadoop/mapred/DirectFileOutputCommitter.java

```diff
@@ -0,0 +1,88 @@
+package org.apache.hadoop.mapred;
```

Comment:
The problem is, some of the helpers are package level scoped. Trying to
move them out creates unnecessary duplication of code.

On Thu, Jul 30, 2015 at 11:54 AM, Ashwanth Kumar <notifications@github.com>
wrote:

> In
> hadoop-mr-s3/src/main/java/org/apache/hadoop/mapred/DirectFileOutputCommitter.java
> <https://github.com/ind9/hadoop-s3/pull/1#discussion_r35841815>:
>
> > @@ -0,0 +1,88 @@
> > +package org.apache.hadoop.mapred;
>
> Can we move this outside org.apache.hadoop.mapred? It really helps while
> debugging (esp. on cases like Qubole where they might have their own
> version and we don't know which class implementation got picked). Sorry, if
> I sound little paranoid.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ind9/hadoop-s3/pull/1/files#r35841815>.
>